### ISSUE TITLE
[Submission]: 2023 Observation Synthesis

### ISSUE THREAD

**Issue by khyde** [NOAA-EDAB/catalog#100]
### Primary Contact

kimberly.hyde@noaa.gov

### Secondary Contact

nefsc.soe.leads@noaa.gov

### Data Name (This will be the displayed title in Catalog)

2023 Observation Synthesis

### Indicator Name (as exists in ecodata)

ESP_gsi

### Family (Which group is this indicator associated with?)

- [X] Oceanographic
- [X] Habitat
- [X] Lower trophic levels
- [ ] Megafauna
- [ ] Social
- [ ] Economic

### Data Description

Synthesis of multiple anomalous and extreme conditions observed in 2023 that should be noted and considered in future analyses.

### Introduction to Indicator (Please explain your indicator)

Multiple anomalous conditions and extreme events were observed in 2023 that could have brief local effects and/or widespread long-term ecosystem, fishery and management implications. This section intends to provide a record of these observations, the implications they may have for other ecosystem processes, and a reflection on how they fit into our understanding of the ecosystem. Many of these observations are being actively studied but should be noted and considered in future analyses and management decisions.

### Key Results and Visualization

TBD

### Implications

TBD

### Spatial Scale

NES

### Temporal Scale

2023

### Synthesis Theme

- [X] Multiple System Drivers
- [ ] Regime Shifts
- [ ] Ecosystem Reorganization

### Define Variables

NA

### Indicator Category

- [ ] Published Methods
- [ ] Extensive analysis, not yet published
- [ ] Syntheses of published information
- [ ] Database pull
- [ ] Database pull with analysis
- [X] Other

### If other, please specify indicator category

Synthesis of observations

### Data Contributors

Kimberly Hyde, Sarah Gaichas, Joe Caracappa

### Point(s) of Contact

kimberly.hyde@noaa.gov

### Affiliation

NEFSC

### Public Availability

Source data are publicly available.

**Comment by jcaracappa1** [comment]
bottom_temp_threshold_17_19_Elephant_Trunk.png should be added to this page

**Comment by andybeet** [comment]
@BBeltz1 This needs an "indicator name" 
`obseravtional_synthesis`?

**Comment by khyde** [comment]
I updated some of the text a bit (fixed some typos, added some hyperlinks and added a little bit of content). 

Here is the link to the updated SST image with the Gulf Stream. https://drive.google.com/file/d/1ObYblh4-bCXCvIqqyJdWtpJ5qXar42Tb/view

Please update the SST caption to read: "Weekly mean sea surface temperature (October 8-10, 2023) with the long-term mean Gulf Stream position.  Red lines represent the 26oC (78.8 oF) temperature contour." and add a space after the caption so it is separate from the following paragraph.

Please update the SST days caption to read: "The number of days in 2022 that exceeded the scallop stress temperature of XX" and add a space after the caption so it is separate from the following paragraph. 
(@jcaracappa1 what is the temperature threshold used in this plot? 18oC?)

Please update the CHL caption to read: "The chlorophyll anomaly for June 2023.  Chlorophyll concentrations in the Gulf of Maine were 5-10 times greater than the long-term June average." and add a space after the caption so it is separate from the following paragraph. 

Thanks!

**Comment by khyde** [comment]
Also, here is the alt text I wrote for the SST and CHL images.

CHL image alt text: A chlorophyll anomaly map from June 2023 for the Northeast shelf ecosystem showing very high chlorophyll anomalies in the Gulf of Maine and outer edge of Georges Bank, moderate above average anomalies in the eastern half of the Mid-Atlantic and moderate below average anomalies in the western Mid-Atlantic.

SST image alt text: A sea surface temperature map from October 2023 for the Northeast shelf ecosystem showing colder temperatures (12-16 degrees Celsius) in the Gulf of Maine and Georges Bank, moderate temperatures (18-22 degrees Celsius) in the western Mid-Atlantic and temperatures above 23 degrees Celsius) in the southeastern Mid-Atlantic.  The Gulf Stream is almost parallel with the continental shelf break and north of the mean Gulf Stream path.

**Comment by jcaracappa1** [comment]
@BBeltz1 The temperature used in the thermal habitat gridded map are 17-19 deg C. Here's the caption for that figure in the report: 
"The number of days in 2022 where bottom temperature was between 17 and 19 ℃  (sressful thermal temeperaturs for sea scallops) in each GLORYS grid cell. The gray lines show the sea scallop estimation areas, with the Elephant Trunk region highlighted in black lines."

**Comment by sgaichas** [comment]
Last year the information on coastal hypoxia was in the ocean acidification page; this year that page has updated but we should keep the 2023 content and add it here so we can point to it. I suggest adding this text (taken from the older edit of the ocean acidification page)

>Summer 2023 multi-stressor event in the Mid-Atlantic

>From August-September 2023, much of the bottom water sampled from Sandy Hook, New Jersey south to Tuckerton, New Jersey and from nearshore (15 meter water depth) to deeper depths (60 meter water depth), exhibited dissolved oxygen concentrations less than 5 mg/L and pH values less than 7.75. Coast-wide, hypoxic levels of dissolved oxygen (concentrations < 3 mg/L) were observed at shallower, more inshore locations. Some of these locations were shallow enough to be observed in glider ru40, but much of these hypoxic levels were observed in a shallow glider (ru28, with a 50 m depth capacity) deployed at the same time. In addition to low pH measured in bottom waters by glider ru39, which is indicative of ocean acidification, aragonite saturation state (Omega) was calculated to be < 1 in several locations. Normal, more optimal levels in seawater typically include dissolved oxygen concentrations > 7 mg/liter, pH of ~8.1, and aragonite saturation states > 3.

>During the time when low dissolved oxygen and pH/aragonite saturation state were observed, numerous mortalities of fish, lobsters and crabs within the sampling area were reported. The mortalities were observed in bottom waters primarily off the coast of Monmouth and Ocean Counties and included the Mud Hole, as far east as Lillian wreck, and southward in Sea Girt and Axel Carlson Reefs and the surrounding areas. Mortalities were reported for American lobsters, Jonah crab, Atlantic rock crab, spider crabs, black sea bass, and tautog not only in pots where trapped organisms would not have been able to escape poor conditions, but also on the open bottom. This observation suggests that if low dissolved oxygen and/or pH were indeed the culprit for these reported mortalities, the area may have been extensive enough that they could not escape in time.

>Healthy dissolved oxygen, pH, and aragonite saturation state levels were restored in bottom waters temporarily during the offshore passage of Hurricane Lee around 09/16, and for the remainder of the sampling after a strong hybrid storm that started on 09/24 and drove the eventual fall water column turnover on 09/26.

**Comment by sgaichas** [comment]
also this one is currently called "observation_synthesis" but we'll need to rename since there will be one for each year

**Comment by BBeltz1** [comment]
Yes I noticed this yesterday too. I will be adding the year to the name of this and future observation synthesis pages.

**Comment by BBeltz1** [comment]
> Last year the information on coastal hypoxia was in the ocean acidification page; this year that page has updated but we should keep the 2023 content and add it here so we can point to it. I suggest adding this text (taken from the older edit of the ocean acidification page)
> 
> > Summer 2023 multi-stressor event in the Mid-Atlantic
> 
> > From August-September 2023, much of the bottom water sampled from Sandy Hook, New Jersey south to Tuckerton, New Jersey and from nearshore (15 meter water depth) to deeper depths (60 meter water depth), exhibited dissolved oxygen concentrations less than 5 mg/L and pH values less than 7.75. Coast-wide, hypoxic levels of dissolved oxygen (concentrations < 3 mg/L) were observed at shallower, more inshore locations. Some of these locations were shallow enough to be observed in glider ru40, but much of these hypoxic levels were observed in a shallow glider (ru28, with a 50 m depth capacity) deployed at the same time. In addition to low pH measured in bottom waters by glider ru39, which is indicative of ocean acidification, aragonite saturation state (Omega) was calculated to be < 1 in several locations. Normal, more optimal levels in seawater typically include dissolved oxygen concentrations > 7 mg/liter, pH of ~8.1, and aragonite saturation states > 3.
> 
> > During the time when low dissolved oxygen and pH/aragonite saturation state were observed, numerous mortalities of fish, lobsters and crabs within the sampling area were reported. The mortalities were observed in bottom waters primarily off the coast of Monmouth and Ocean Counties and included the Mud Hole, as far east as Lillian wreck, and southward in Sea Girt and Axel Carlson Reefs and the surrounding areas. Mortalities were reported for American lobsters, Jonah crab, Atlantic rock crab, spider crabs, black sea bass, and tautog not only in pots where trapped organisms would not have been able to escape poor conditions, but also on the open bottom. This observation suggests that if low dissolved oxygen and/or pH were indeed the culprit for these reported mortalities, the area may have been extensive enough that they could not escape in time.
> 
> > Healthy dissolved oxygen, pH, and aragonite saturation state levels were restored in bottom waters temporarily during the offshore passage of Hurricane Lee around 09/16, and for the remainder of the sampling after a strong hybrid storm that started on 09/24 and drove the eventual fall water column turnover on 09/26.

Text has been integrated.

**Comment by khyde** [comment]
@BBeltz1 Please add the following sentence to the end of the tripos paragraph in the Regional/Coastal Phenomena section (paragraph 3).

See [this report](https://sea-scope.github.io/Tripos/report.html) for more information on how the Tripos bloom was tracked by a network of regional collaborators and the subsequent workshop to advance an understanding of the bloom.
